src/pages/LandingPage/LandingPage.tsx

```diff
@@ -71,7 +71,7 @@ const LandingPage: React.FC = () => {
       title: 'Reddit',
     },
     {
-      link: 'https://discord.com/invite/XJTM7FV88Y',
+      link: 'https://discord.gg/dSMd7AFH36',
```

Comment:
This link is wrong the link should not be an invite from a specific user.
The invite should be from discord